install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -146,15 +146,11 @@ data:
           runAsUser: 1337
           {{ "[[- end ]]" }}
         resources:
-          {{ "[[ if (isset .ObjectMeta.Annotations \"sidecar.istio.io/proxyCPU\") -]]" }}
           requests:
-            cpu: {{ "\"[[ index .ObjectMeta.Annotations \"sidecar.istio.io/proxyCPU\" ]]\"" }}
-            memory: {{ "\"[[ index .ObjectMeta.Annotations \"sidecar.istio.io/proxyMemory\" ]]\"" }}
-        {{ "[[ else -]]" }}
-{{- if .Values.global.proxy.resources }}
-{{ toYaml .Values.global.proxy.resources | indent 10 }}
-{{- end }}
-        {{ "[[ end -]]" }}
+            cpu: {{ "[[ annotation .ObjectMeta `sidecar.istio.io/proxyCPU` " }} "{{ .Values.global.proxy.resources.requests.cpu }}" {{ " ]]" }}
+            {{ "[[ if ne (annotation .ObjectMeta `sidecar.istio.io/proxyMemory` \"\") \"\" -]]" }}
```

Comment:
do we want to establish a default value for memory in the configmap as well?

@ostromart thoughts?

Reply:
Yes, I also think 10m might be too low (below). @costinm has been doing scale testing so he probably has the best idea for good default values here.

Reply:
I think we should also need to implement the ability to config the resource limit? It's related to the pod QoS class in k8s.